qa/tasks/cephfs/test_volumes.py

```diff
@@ -1654,8 +1654,8 @@ def test_subvolume_group_resize_infinite_size_future_writes(self):
         subvolpath = self._get_subvolume_path(self.volname, subvolname, group_name=group)
         self.assertNotEqual(subvolpath, None)
 
-        # create 4 files of 1MB
-        self._do_subvolume_io(subvolname, subvolume_group=group, number_of_files=4)
+        # create 7 files of 1MB
```

Comment:
any reason to bump up number of files from 4 to 7?

Reply:
Yeah, because the `osize` value changed from `4MB` --> `8MB`.